### PR TITLE
Update install.md for ios: Mapbox Maps SDK

### PR DESCRIPTION
**Pull request by mzu**
Following ticket https://github.com/react-native-mapbox-gl/maps/issues/1103 to explain how to use a custom version of the Mapbox Maps SDK for iOS.

### CODE REVIEW

**Comment by ferdicus** [comment]
Hey @mzu, thanks for this 👍🏿 

One suggestion: Instead of linking to the PR, could you instead have the instructions mirrored in the installation guide please.
I know it's basically duplicating the info - however I feel like the guide should rather be the nexus for such info instead of linking to a PR. 

Thanks in advance 🙇🏿

**Comment by mzu** [comment]
Hi @ferdicus. 

Added the instructions. Hope the formatting looks ok.

File: ios/install.md [hunk 1]
```diff
@@ -31,4 +31,39 @@ Mapbox normally [requires](https://github.com/mapbox/mapbox-gl-native-ios/issues
...
+
+Add the following to you `ios/Podfile`:
+
+```ruby
+$ReactNativeMapboxGLIOSVersion = '~> 6.1'
+```
+
+Check the current version of the SDK [here](https://docs.mapbox.com/ios/maps/overview/).
+
+### Deprecated version
```
Comment (mfazekas) [inline ios/install.md:48]: Can we remove the deprecated section?!

Comment (mfazekas) [reply ios/install.md:48]: Also we need to at least a note on using 6.0 and later.

https://github.com/react-native-mapbox-gl/maps/pull/1044#issue-493540536

Comment (mzu) [reply ios/install.md]: > Can we remove the deprecated section?!

That's done now.


> Also we need to at least a note on using 6.0 and later.
> 
> [#1044 (comment)](https://github.com/react-native-mapbox-gl/maps/pull/1044#issue-493540536)

There is the already existing section "Mapbox Maps SDK > `v6.0.0`". Does this cover it?

**Comment by ferdicus** [comment]
Thanks for this @mzu, merging this for now - we can alter it later.
I feel it's good to have it already included so we can point at it and people can discover it - instead of us redirecting people to comments and PRs 👍🏿

**Pull request merged**